webapp/components/post_view/post_body_additional_content.jsx

```diff
@@ -82,6 +91,24 @@ export default class PostBodyAdditionalContent extends React.PureComponent {
         );
     }
 
+    // when image links are collapsed, check if the link is a valid image url and it is available
+    checkIfLinkImageExists() {
```

Comment:
Function name is confusing at first read. Could it be change to something like: `precheckImageLink`?